test/e2e/etcdbackupserver_e2e_test.go

```diff
@@ -0,0 +1,147 @@
+		require.NoError(t, err)
+	}
+
+	t.Cleanup(func() {
+		cErr := configClient.Backups().Delete(context.Background(), backupCR.Name, metav1.DeleteOptions{})
+		require.NoError(t, cErr)
+	})
+}
+
+func createDefaultBackupCR(schedule, timeZone string) *configv1alpha1.Backup {
+	return &configv1alpha1.Backup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: defaultBackupCRName,
+			Annotations: map[string]string{
+				"default": "true",
+			},
+		},
+		Spec: configv1alpha1.BackupSpec{
+			EtcdBackupSpec: configv1alpha1.EtcdBackupSpec{
+				Schedule: schedule,
+				TimeZone: timeZone,
+				RetentionPolicy: configv1alpha1.RetentionPolicy{
+					RetentionType: configv1alpha1.RetentionTypeNumber,
+				},
+			},
+		},
+	}
+}
+
+func isBackupExist(t *testing.T, node corev1.Node) error {
+	client := framework.NewCoreClient(t)
+	pod := corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      names.SimpleNameGenerator.GenerateName("backup-finder-pod" + "-"),
+			Namespace: OpenShiftEtcdNamespace,
+		},
+		Spec: corev1.PodSpec{
+			NodeName: node.Name,
+			Volumes: []corev1.Volume{
+				{
+					Name: "backup-dir",
+					VolumeSource: corev1.VolumeSource{
+						HostPath: &corev1.HostPathVolumeSource{
+							Path: backupHostPath,
+						},
+					},
+				},
+			},
+			Containers: []corev1.Container{
+				{
+					Name:       "finder",
+					Image:      ShellImage,
+					Command:    []string{"find", "."},
+					WorkingDir: backupHostPath,
+					Resources:  corev1.ResourceRequirements{},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "backup-dir",
+							ReadOnly:  false,
+							MountPath: backupHostPath,
+						},
+					},
+					SecurityContext: &corev1.SecurityContext{Privileged: pointer.Bool(true)},
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyNever,
+		},
+	}
+
+	_, err := client.Pods(OpenShiftEtcdNamespace).Create(context.Background(), &pod, metav1.CreateOptions{})
+	require.NoError(t, err)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
+	defer cancel()
+	err = wait.PollUntilContextCancel(ctx, 10*time.Second, false, func(ctx context.Context) (done bool, err error) {
+		p, err := client.Pods(OpenShiftEtcdNamespace).Get(context.Background(), pod.Name, metav1.GetOptions{})
+		if err != nil {
+			klog.Infof("error while getting finder pod: %v", err)
+			return false, nil
+		}
+
+		if p.Status.Phase == corev1.PodFailed {
+			return true, fmt.Errorf("finder pod failed with status: %s", p.Status.String())
+		}
+
+		return p.Status.Phase == corev1.PodSucceeded, nil
+	})
+	require.NoErrorf(t, err, "waiting for finder pod error")
+
+	logBytes, err := client.Pods(OpenShiftEtcdNamespace).GetLogs(pod.Name, &corev1.PodLogOptions{}).Do(ctx).Raw()
+	require.NoError(t, err)
+
+	files := strings.Split(string(logBytes), "\n")
+	klog.Infof("found files on node [%s]: %v", node.Name, files)
+
+	require.Contains(t, files[2], "snapshot_")
+	require.Contains(t, files[3], "static_kuberesources_")
```

Comment:
@tjungblu  Is this enough assertion ? 

also the cleanUp, any suggestions ?

Reply:
don't we already have assertions from the other backup logic?

Reply:
u mean https://github.com/openshift/cluster-etcd-operator/blob/cf335f20dc98096f110b118bff91d519b1c096c6/test/e2e/backup_test.go#L671 ?

Reply:
i kept this for now, since I dont use a prefix for the backup name, the current assertion relies on the prefix